frontend/src/map/map.jsx

```diff
@@ -345,6 +409,7 @@ export default function MapComponent(props) {
     );
 
     map.on(['click'], async (event) => {
+      // console.log(map.getCoordinateFromPixel(event.pixel));
```

Comment:
Leftover logging?

Reply:
I thought it is somewhat likely I will want to use this again in the future, so decided to leave it in for convenience.